powersimdata/scenario/move.py

```diff
@@ -36,21 +41,11 @@ def move_scenario(self, target="disk", confirm=True):
             raise ValueError("scenario data can only be backed up to disk now")
 
         backup = BackUpDisk(self._data_access, self._scenario_info)
```

Comment:
I think you can pass `self_scenario_info["id"]`. You could declare `sid` before creating this `BackUpDisk` instance and use it for the instantiation